src/components/constants/Routes.js

```diff
@@ -30,4 +30,7 @@ export const MY_PROFILE_PAGE = "/my-profile";
 
 export const UPDATE_STATUS_PAGE = "/update-status";
 
-export const USER_SETTINGS_PAGE = "/user-settings";
\ No newline at end of file
+export const USER_SETTINGS_PAGE = "/user-settings";
+
+// Resources Routes
+export const CREATE_RESOURCES = "/create-resource";
```

Comment:
make sure to update the route name here too, and follow the naming convention
ex: `CREATE_RESOURCE_PAGE`